docs/components/drawer/drawer-demo.js

```diff
@@ -0,0 +1,38 @@
+import Util from '../../_util';
+
+if (document.getElementById('vue-drawerDemo')) {
+    new Vue({
+        el: '#vue-drawerDemo',
+        data: {
+            isScrollable: false,
+        },
+        computed: {
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-disclosure
+                        aria-controls="demo-drawer"
```

Comment:
fix indentation to 2 instead of 4 spaces on lines 13-14

Reply:
indentation fixed